### PR TITLE
Creating login form component in react

### PR DESCRIPTION
**Pull request by MilenaHtec**

### CODE REVIEW

File: frontend/src/Components/Loader/Loader.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			<div className="spinner-border text-primary" style={{ width: "5rem", height: "5rem" }} role="status">
+				<span className="sr-only">Loading...</span>
+			</div>
+
+			{/* This is the optional Loader that we may use. */}
+			{/* <div className="spinner-grow" style={{ width: "5rem", height: "5rem" }} role="status">
+					<span className="sr-only">Loading...</span>
+				</div> */}
+			<br />
+			<div className="text-muted" style={{ fontSize: "20px" }}>
```
Comment (marija-antic) [inline frontend/src/Components/Loader/Loader.jsx]: Avoid inline styling, use classes instead.

File: frontend/src/Components/Loader/Loader.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	return (
+		// Loader animation with Text - Can be changed whenever, however
+		<div className="container-fluid d-flex flex-column justify-content-center align-items-center vh-100">
+			{/* Main Loader Style */}
+			<div className="spinner-border text-primary" style={{ width: "5rem", height: "5rem" }} role="status">
+				<span className="sr-only">Loading...</span>
+			</div>
+
+			{/* This is the optional Loader that we may use. */}
+			{/* <div className="spinner-grow" style={{ width: "5rem", height: "5rem" }} role="status">
```
Comment (marija-antic) [inline frontend/src/Components/Loader/Loader.jsx]: Remove commented code, also review other comments if they're making sense like the one on line 3.

File: frontend/src/index.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import App from "./App";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+	<React.StrictMode>
```
Comment (marija-antic) [inline frontend/src/index.js]: Just be aware that Strict mode has differences in local env and on f.e. prod, as far as I know.

**Comment by MilenaHtec** [comment]
Can we remove Strict mode?
________________________________

File: frontend/src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Loader from "./Components/Loader/Loader";
+
+function App() {
```
Comment (dusantrtica) [inline frontend/src/App.jsx]: I think that we don't need to define functions with a 'function' keyword.
Also return statement is not needed, you can just write something like:
const App = () => <Loader />

File: frontend/src/index.js (hunk 3 above)
Comment (casualme2) [reply frontend/src/index.js]: We will remove it, thanks.

File: frontend/src/App.jsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Loader from "./Components/Loader/Loader";
+
+const App = () => {
```
Comment (marija-antic) [inline frontend/src/App.jsx:4]: this can be even shorter: 
`const App = () => <Loader />;`

File: frontend/src/Components/Loader/Loader.jsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Loader.css";
+
+const Loader = () => {
```
Comment (marija-antic) [inline frontend/src/Components/Loader/Loader.jsx:4]: Same here:
`const Loader = () => (
		<div className="container-fluid d-flex flex-column justify-content-center align-items-center vh-100">
			<div className="spinner-border text-primary loader-circle-size" role="status">
				<span className="sr-only">Loading...</span>
			</div>
			<br />
			<div className="text-muted loader-text-size">Loading your data...</div>
		</div>
	);`

**Review by marija-antic — APPROVED** [review]

**Pull request merged**